apps/condo/domains/miniapp/schema/SendB2CAppPushMessageService.js

```diff
@@ -172,6 +173,14 @@ const SendB2CAppPushMessageService = new GQLCustomSchema('SendB2CAppPushMessageS
                     },
                 }
 
+                if (voipType) {
```

Comment:
maybe check that the `type` is equal to `VOIP_INCOMING_CALL_MESSAGE`?

Reply:
this is to determine the integration there may be. 'sip' or 'basIp'. So far only 'sip'